klaus/highlighting.py

```diff
@@ -34,17 +36,18 @@ def _format_lines(self, tokensource):
 
     def _lookup_ctag(self, token):
         matches = list(self._get_all_ctags_matches(token))
-        best_matches = self.get_best_ctags_matches(matches)
-        if not best_matches:
+        try:
+            best_match = next(self.get_best_ctags_matches(matches))
```

Comment:
Well using `next()` will be faster since it means we can stop before we have the complete list. To be honest, I'd have used it for `matches` too :)
But as it's your module, I'll use the list approach. I'll repush in a few minutes.